src/components/HomeContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from 'react';
+
+import { QuestionContext } from '../contexts/QuestionsContext';
+import { ResponseContext } from '../contexts/ResponseContext';
+import HomeScreen from '../screens/HomeScreen';
+
+export interface BadgeTitleRewardText {
+  badgeTitle: string;
+  rewardText: string;
+}
+
+export const HomeContainer = () => {
```

Comment:
I think it would be helpful to either annotate this component's function with a high level overview of _how_ the most recent badges are determined (i.e. get response list -> filter out positive responses -> determine badge title + reward text) or annotate each step of the process inside the function. The home screen will receive major updates for the Fall version, so the more comments, the better.